guides/source/active_storage_overview.md

```diff
@@ -246,6 +246,25 @@ google:
   cache_control: "public, max-age=3600"
 ```
 
+Optionally use [IAM](https://cloud.google.com/storage/docs/access-control/signed-urls) instead of the `credentials` when signing URLs.
```

Comment:
Should this link to the specific use of IAM on that page? Also, is there some GCP documentation we can link to that would inform a user of when they would want to use IAM?

```suggestion
Optionally use [IAM](https://cloud.google.com/storage/docs/access-control/signed-urls#signing-iam) instead of the `credentials` when signing URLs.
```

Reply:
Best I found is https://cloud.google.com/blog/products/containers-kubernetes/introducing-workload-identity-better-authentication-for-your-gke-applications, I'll add a mention to it.